### DISCUSSION TITLE
Is there a way to bundle a sub path config using tsup?

### DISCUSSION THREAD

**Discussion by thedanchez** [egoist/tsup#830] [Q&A]
I'm trying to package up a library that contains a sub-path export (`dist/subPath/`) alongside the main `dist/index.jsx` export. Does tsup support this use-case?

**Comment by wottpal** [comment]
> I'm trying to package up a library that contains a sub-path export (`dist/subPath/`) alongside the main `dist/index.jsx` export. Does tsup support this use-case?

Did you find a solution to this?
